wordpress-mobile/WordPress-iOS
# Media Library: No Results View

This PR adds a No Results view to the media library when the user's media library is empty. 

![noresults](https://cloud.githubusercontent.com/assets/4780/24247566/2a53273a-0fc4-11e7-9139-6f23fcf3b2aa.png)

Note that the Upload button in the no results view is currently nonfunctional.

To test:

* Ensure the no results view shows correctly when there are no items, and no search results.
* Check it appears correctly if you delete the last item from the web, or add a new item to the web.
* Check the collection view scroll insets still work correctly with the keyboard, as I switched things to use a stack view.

Needs review: @kurzee
